.eslintrc.json

```diff
@@ -42,7 +42,6 @@
         "no-shadow": "off",
         "@typescript-eslint/no-var-requires": "off",
         "react-hooks/rules-of-hooks": "error",
-        "react-hooks/exhaustive-deps": "warn",

```

Comment:
Please, return this rule and don't disable it. It's quite important and helps avoid bugs.